.github/workflows/run_cts_tests.yaml

```diff
@@ -139,9 +139,9 @@ jobs:
           docker-compose -f .ci/compatibility_tests/docker-compose-mysql-tls.yaml up -d
           echo '\n' >> apps/emqx_auth_mongo/etc/emqx_auth_mysql.conf
           echo 'auth.mysql.ssl = on' >> apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
-          echo "auth.mysql.ssl.cafile = /emqx/apps/emqx_auth_mysql/test/emqx_auth_mysql_SUITE_data/ca.pem" >> apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
-          echo "auth.mysql.ssl.certfile = /emqx/apps/emqx_auth_mysql/test/emqx_auth_mysql_SUITE_data/client-cert.pem" >> apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
-          echo "auth.mysql.ssl.keyfile =  /emqx/apps/emqx_auth_mysql/test/emqx_auth_mysql_SUITE_data/client-key.pem" >> apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
+          echo "auth.mysql.ssl.cafile = \"/emqx/apps/emqx_auth_mysql/test/emqx_auth_mysql_SUITE_data/ca.pem\"" >> apps/emqx_auth_mysql/etc/emqx_auth_mysql.conf
```

Comment:
there is no need to escape double quotes if we change to quote the `echo` content with single quotes.

Reply:
nitpicking, no need to fix. especially since we have bash string substitution. e.g. line 154